types.tsx

```diff
@@ -36,7 +35,7 @@ export type RootStackScreenProps<Screen extends keyof RootStackParamList> = Nati
 >;
 
 export type RootTabParamList = {
-  Expenses: undefined;
+  Expenses?: { refresh?: boolean };
   Budget: undefined;
```

Comment:
Can you add refresh boolean to all screens

Reply:
Each screen should opt into having a refresh param on a case by case basis, depending on whether that screen fetches data which can be updated.